src/extlibs/enlsvg/Pathfinding/Fraction.h

```
@@ -0,0 +1,190 @@
+#ifndef FRACTION_H
+#define FRACTION_H
+
+#include <sstream>
+#include <string>
+using namespace std;
+
+namespace Pathfinding
+{
+    struct Fraction
+    {
+        long n, d;
+
+        inline Fraction() {}
+
+        inline Fraction(long n, long d)
+        {
+            init(n, d);
+        }
+
+        inline Fraction(long n) : n(n), d(1) {}
+
+        inline void init(long n)
+        {
+            this->n = n;
+            this->d = 1;
+        }
+
+        inline void init(long n, long d)
+        {
+            /*if (d == 0) {
+                cout << "\nERROR: Zero denominator: " << n << + "/" << d << "\n";
+                return;
+            }*/
```

Curious if this should remain commented out? Should we throw an exception?

---

I didn't comment this out. This is out of the box from their git, and I didn't want to touch it

---

If we're planning to eventually move these files out of `extlib` and it's not used in the implementation, I think we're safe to remove the commented out code then.

---

agreed

---

removed